src/foundation.cs

```diff
@@ -3258,6 +3264,10 @@ public interface NSExpression : NSSecureCoding, NSCopying {
 		[Export ("falseExpression")]
 		NSExpression FalseExpression { get; }
 
+		[Mac(10,6)]
```

Comment:
It's the minimum supported (or it might be 10.7 now) so you do not have to add the metadata.